src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private ColorSupplier randomColor = new ColorSupplier();
+    private Ball randomBall = new Ball();
```

Comment:
```suggestion
    private ColorSupplier randomColor = new ColorSupplier();
    private Random random = new Random();
```
ball should be a local variable - because you need a new one each time this method is called